content/ch-algorithms/superdense-coding.ipynb

```diff
@@ -142,15 +142,14 @@
     "# Importing everything\n",
     "from qiskit import *\n",
     "from qiskit.visualization import plot_histogram\n",
-    "# For Jupyter Notebooks, change the settings to get nicer images\n",
-    "%config InlineBackend.figure_format = 'svg'"
+    "# For Jupyter Notebooks, change the settings to get nicer images"
```

Comment:
If removing the `%config` line, we should also remove this comment.

Reply:
My apologies @frankharkins. I forgot about the comment.